assets/js/modules/reader-revenue-manager/datastore/publications.test.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * `modules/reader-revenue-manager` data store: publications tests.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import {
+	createTestRegistry,
+	untilResolved,
+} from '../../../../../tests/js/utils';
+import * as fixtures from './__fixtures__';
+import { MODULES_READER_REVENUE_MANAGER } from './constants';
+
+describe( 'modules/reader-revenue-manager publications', () => {
+	let registry;
+
+	const publicationsEndpoint = new RegExp(
+		'^/google-site-kit/v1/modules/reader-revenue-manager/data/publications'
+	);
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	describe( 'selectors', () => {
+		describe( 'getPublications', () => {
+			it( 'should use a resolver to make a network request', async () => {
+				fetchMock.get( publicationsEndpoint, {
+					body: fixtures.publications,
+					status: 200,
+				} );
+
+				const initialProperties = registry
```

Comment:
The variable name here appears wrong.

```suggestion
				const initialPublications = registry
```